ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ActiveContracts.scala

```diff
@@ -129,7 +147,8 @@ class ActiveContractsManager[ACS](initialState: => ACS)(implicit ACS: ACS => Act
                     })
                   } else {
                     acc
-                  })
+                  }),
+                  parties = parties.union(nodeParties)
                 )
               case nlkup: N.NodeLookupByKey.WithTxValue[AbsoluteContractId] =>
```

Comment:
Do we not need to implicitly create the maintainer parties of the key as well?

Reply:
Good catch.